dcompiler.dd

```diff
@@ -603,6 +603,10 @@ dmd -cov -unittest myprog.d
         implement http://wiki.dlang.org/DIP25 (experimental)
       )
 
+      $(SWITCH $(SWNAME -dip1000),
+        implement http://wiki.dlang.org/DIP1000 (experimental)
```

Comment:
that should be sth. like `$(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1000.md, DIP1000)`

Reply:
Fixed it myself.

Reply:
Thanks - I got the wrong link from stable `dmd --help` (fixed in master) :-/. Also do you know what's happening about ddoc automatically recognising links? (The `-dip25` link is plain text too).